src/Sylius/Behat/Service/DriverHelper.php

```diff
@@ -14,14 +14,15 @@
 namespace Sylius\Behat\Service;
 
 use Behat\Mink\Driver\DriverInterface;
+use Behat\Mink\Driver\PantherDriver;
 use Behat\Mink\Driver\Selenium2Driver;
 use DMore\ChromeDriver\ChromeDriver;
 
 abstract class DriverHelper
 {
     public static function isJavascript(DriverInterface $driver): bool
     {
-        return $driver instanceof Selenium2Driver || $driver instanceof ChromeDriver;
+        return $driver instanceof Selenium2Driver || $driver instanceof ChromeDriver || $driver instanceof PantherDriver;
```

Comment:
I think we already talked about it in another PR did by @loic425 a long time ago, we can create a specific service for this, alowing other exotic driver to be able to hook into this by decorating this new service.